modules/orchestrator/proc-install-components-for-orchestrator-plugin.adoc

```diff
@@ -71,4 +66,9 @@ $ ./plugin-infra.sh
 ----
 
 .{product-very-short} Orchestrator Infra Helm Chart
```

Comment:
The complete name of this chart (as displayed in the OCP web console) is: "Orchestrator Infrastructure for Red Hat Developer Hub" Helm Chart
https://github.com/redhat-developer/rhdh-chart/blob/release-1.7/charts/orchestrator-infra/Chart.yaml#L2